README.md

```diff
@@ -1,119 +1,52 @@
-![](./docs/source/.static/img/nucypher.png)
+![](nucypher.png)
 
-*A decentralized cryptological network offering accessible, intuitive, and extensible runtimes and interfaces for secrets management and dynamic access control.*
+*A decentralized cryptological network offering accessible, intuitive, and extensible runtimes and interfaces for
```

Comment:
Nit-picking, but do we still want to call ourselves a 'cryptological network'? Since NuCypher is a contributor to a bigger network with services that aren't exclusively cryptographic primitives (even if they're based on primitives, that's not how they're consumed), and also 'cryptological' implies that there's both a cryptographic and cryptanalysis component to the service (which would make more sense if we were white hats proving vulnerabilities in SGX or something, rather than just loosely that we analyze the security of our own offerings).